rc/tools/git.kak

```diff
@@ -1,6 +1,18 @@
 declare-option -docstring "name of the client in which documentation is to be displayed" \
     str docsclient
 
+declare-option -docstring "diff added character" \
+    str diff_add_char "▏"
+
+declare-option -docstring "diff modified character" \
+    str diff_mod_char "▏"
+
+declare-option -docstring "diff deleted character" \
+    str diff_del_char "▁"
+
+declare-option -docstring "diff top deleted character" \
+    str diff_top_char "‾"
```

Comment:
Not too sure about the names, maybe `git_diff_` prefix would be more appropriate.